src/components/PriceCheckForm.js

```diff
@@ -1,33 +1,70 @@
 import React, { Component } from 'react'
+import axios from 'axios';
+import { categories } from '../utils/Constants.js';
+
 
 class PriceCheckForm extends Component {
     constructor(props){
         super(props)
         this.state = {
-            postcode: ""
+            postcode: "",
+            selectedCategory: "",
+            searchString: ""
         }
 
         this.handleSubmit = this.handleSubmit.bind(this);
-        this.handleChange = this.handleChange.bind(this);
-       
+        this.handlePostcodeChange = this.handlePostcodeChange.bind(this);
+        this.handleCategoryChange = this.handleCategoryChange.bind(this);
+        this.handleSearchStringChange = this.handleSearchStringChange.bind(this);
+        this.fetchSuppliers = this.fetchSuppliers.bind(this)
+    }
+
+    fetchSuppliers = (postcode) => {
+        return axios.get(`https://www.foodbomb.com.au/supplier-service/postcode/${postcode}/suppliers`)
```

Comment:
I don’t think this syntax will do what you’re expecting, did you want to return the supplierIds or the promise from the axios cal ?